### ISSUE TITLE
Dataset problem

### ISSUE THREAD

**Issue by Jiawen-huang** [cyvius96/few-shot-meta-baseline#13]
the miniImageNet dataset used in your paper:
miniImageNet_category_split_train_phase_train.pickle
miniImageNet_category_split_train_phase_val.pickle
miniImageNet_category_split_train_phase_test.pickle
miniImageNet_category_split_val.pickle
miniImageNet_category_split_test.pickle
I know ：
①miniImageNet_category_split_val.pickle  is the validation set of miniImageNet which has 16 classes and Each class has 600 images
②miniImageNet_category_split_test.pickle a test set of miniImageNet which has 20 classes and Each class has 600 images.
③miniImageNet_category_split_train_phase_train.pickle a train set of miniImageNet which has 64 classes and Each class has 600 images.
but I don't know the detail of miniImageNet_category_split_train_phase_val.pickle and
miniImageNet_category_split_train_phase_test.pickle.??????????????
Is there any overlap between these two data sets and the data inminiImageNet_category_split_train_phase_train.pickle or something else?

**Comment by cyvius96** [comment]
Hi. This miniImageNet is downloaded from the project: https://github.com/gidariss/FewShotWithoutForgetting.

The train_phase_train/val/test are used for testing base classes, they are all from the 64 base classes, and the images in each set does not overlap to each other.

**Comment by Jiawen-huang** [comment]
I used this data, and I ran your experiment. I found that  train_phase_train has 64*600=38400 images which are from 64  base classes. According to the instructions which says base class has 64 classes and each class has 600 images, I think train_phase_train has all the training data from the 64  base classes.
However, I found that The train_phase_val has 18748 images which are from 64  base classes. 
So I've been wondering whether the images in train_phase_train/val/test may be overlap  each other or Additional data is used???????
